doc/versionHistory.md

```diff
@@ -1,5 +1,12 @@
 # Version History
 
+0.4.2
+
+- Replace **General Error Handler.vi** with **Unbundle By Name** function to create the string with error code and error explanation in **TcpServerBase.clearErrorAndSendExplnation.vi.**
```

Comment:
error code and error explanation --> error code and error source

Reply:
Fixed.